packages/admin-ui-docs/docs/components/set.md

```diff
@@ -5,7 +5,13 @@ path: /set/
 
 # Set
 
-Our design system comes with a set of layout components to help you build common layouts inside your application. `Set` represents a set of components, that automatically space out the components.
+Our design system comes with a set of layout components to help you build common layouts inside your application. `Set` represents a set of components, it automatically spaces out the components.
```

Comment:
I'll use `child components` because "child component" is used in the react community. I'm basing the plural form on this https://ell.stackexchange.com/questions/234089/is-it-child-elements-or-children-elements